src/main/java/ca/vapurrmaid/discretemathapplications/domain/computation/PrimeFactorizationResult.java

```diff
@@ -0,0 +1,18 @@
+package ca.vapurrmaid.discretemathapplications.domain.computation;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author vapurrmaid
+ */
+public class PrimeFactorizationResult extends ComputationalResult {
+
+    private final Map<Integer, Integer> primeFactors;
```

Comment:
`Map<Integer, Integer>` is a tally of each prime.

Example, `2 2 5` would map to:

```
{ 2: 2, 5: 1 }
```